manifests/t/Tencent/QQ/9.4.9.27849/Tencent.QQ.locale.en-US.yaml

```diff
@@ -0,0 +1,22 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.0.0.schema.json
+
+PackageIdentifier: Tencent.QQ
+PackageVersion: 9.4.9.27849
+PackageLocale: en-US
+Publisher: Tencent Technology (Shenzhen) Co. Ltd.
```

Comment:
![image](https://user-images.githubusercontent.com/52826317/130099010-12cca8bc-814c-4524-a268-99dd4ee0d27e.png)

```suggestion
Publisher: 腾讯科技(深圳)有限公司
```

Reply:
Using Chinese in en-US manifest?

Reply:
Correct. It needs to match the ARP - [schema/1.0.0/defaultLocate#publisher](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/defaultLocale.md#publisher):

> Note: With the 1.0 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package when it has been installed. The impact is associated with winget upgrade and winget list